swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -612,85 +519,42 @@ function getEmailContent(int appointmentNumber, Appointment appointment, Payment
 
 ### Step 3: Build and run the service
 
-You can run this service by navigating to the project root and using the `bal run` command.
-
-```
-sending-emails-from-a-service$ bal run
-Compiling source
-        integration_tutorials/sending_emails_from_a_service:0.1.0
+![Run the service](/learn/images/integration-tutorials/sending-emails-from-a-service/run_the_service.gif)
 
-Running executable
-```
+> **Note:**
+> Alternatively, you can run this service by navigating to the project root and using the `bal run` command.
+>
+> ```
+> sending-a-message-to-a-service$ bal run
+> Compiling source
+>         integration_tutorials/sending_a_message_to_a_service:0.1.0
+>
+> Running executable
+> ```
 
 ### Step 4: Try out the use case
 
 Let's test the use case by sending a request to the service.
 
 #### Start the backend service
 
-Download the JAR file for the [backend service](https://github.com/ballerina-guides/integration-tutorials/blob/main/backends/hospital-service/hospitalservice.jar) and execute the following command to start the service:
+Download the JAR file for the [backend service](https://github.com/ballerina-guides/integration-tutorials/blob/main/backends/hospital-service/hospitalservice.jar) and execute the following command to start the service.
 
 ```
-bal run hospitalservice.jar
+$ bal run hospitalservice.jar
 ```
 
 #### Send a request
 
-Let's send a request to the service using cURL as follows.
+Use the [Try it](https://wso2.com/ballerina/vscode/docs/try-the-services/try-http-services/) feature to send a request to the service. Specify the port used in the service and use `surgery` as the path parameter.
 
```

Comment:
Since this is a POST request, we have to include the sample request payload here.